docs/FAQ.md

```diff
@@ -0,0 +1,13 @@
+# Frequently Asked Questions
+
+## Why the binary name is ocm-backplane ?
+With the name `ocm-*`, the binary can act as an [OCM plugin](https://github.com/openshift-online/ocm-cli/blob/119e9d2e8af0ddf5db4b0888b5ceadb300768885/README.md#extend-ocm-with-plugins).
+
+The logic behind OCM plugin:
+- The user runs `ocm backplane`.
+- `ocm` doesn't have the `backplane` subcommand.
+- `ocm` lookup the PATH and find the `ocm-backplane` binary.
+- `ocm` calls the `ocm-backplane` binary.
+
+## Does ocm-backplane depend on a specific OCM binary version?
+No. The `ocm-backplane` is an independent binary. It is fully functional without the `ocm` binary. The interaction with OCM api will be handled by ocm-sdk, which is defined in the go.mod of ocm-backplane.
```

Comment:
```suggestion
`ocm-backplane` is not a standalone binary. It is functional even without the `ocm` binaries. The interaction with OCM API will be handled by OCM SDK, which is defined as a dependency of the project.
```

Reply:
Thanks @supreeth7 !   Did you mean `not a standalone` or `a standalone`?

Reply:
Right, my bad, ~not~